docs/modules/generic_send.md

```diff
@@ -0,0 +1,77 @@
+# Generic Send (generic_send.py)
```

Comment:
IMHO would be better to describe generic_sender (which does pretty much the exact same, but should replace the generic_send in the future)